Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/FelipeDocil/SwiftGherkin", .branch("support_comments")),
```

Comment:
```suggestion
        .package(url: "https://github.com/iainsmith/SwiftGherkin", .exact("0.2.0")),
```

That solves issue #8

Reply:
Also, this drops Swift 4.1 and adds support to Swift 5, anything should change since my branch already was supporting Swift 5, but I'd test it again just to make sure everything is working.

Reply:
`swift-tools-version:5.0` this already uses Swift 5.0, then should be fine